docs/aws/README.md

```diff
@@ -1,11 +1,31 @@
-# AWS
+# Getting Started with Karpenter on AWS
 
-This guide will provide a complete Karpenter installation for AWS.
-These steps are opinionated and may need to be adapted for your use case.
+Karpenter automatically provisions new nodes in response to unschedulable  pods. Karpenter does this by observing events within the kubernetes cluster, and then sending commands to the underlying cloud provider. 
```

Comment:
Can we try to stick with symantic line breaks in the markdown? It makes reviewing diffs and reorganizing content easier. https://sembr.org/
The main thing is to put each sentence on it's own line. Markdown will combine the lines back into a paragraph but diffs are easier to see what changed and it's easier to leave comments.

On that note the first sentence in this paragraph has an extra space after unschedulable.

Reply:
ok I fixed the extra space. 

Kubernetes uses either "gqq" in vi (wrap long lines), or long lines. I'd rather stick with one of the two existing standards in that space. 

I will not use semantic line breaks. 

First, "A semantic line break should occur after an independent clause as punctuated by a comma (,), semicolon (;), colon (:), or em dash (—)."

This rule is difficult to implement (can't be automated), and doesn't work nicely with soft wrap in vscode. 

Second, it fails in the goal of making git diffs readable. Small changes a slightly more readable, but the benefits are lost when paragraphs are revised.

Reply:
I disagree it fails to make diffs readable, but I'm not going to nit over formatting if not everyone wants to use it. I'll keep writing with sembr on sentence breaks as it has saved me a lot of time for content management and PRs in the past.